### ISSUE TITLE
[a11y] Add external link icon to the links on the confirmation page too

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#555]
Currently there is no external link icon:
<img width="778" alt="Screen Shot 2020-05-28 at 3 03 38 PM" src="https://user-images.githubusercontent.com/2560618/83313071-892abf80-a1c9-11ea-8930-0dc0cf5886c5.png">

We want to follow recommended practices and stay consistent with other links within the website to include the icon:
![Screen Shot 2020-05-28 at 2.35.34 PM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/79d998b6-8b1e-4a5b-9d39-f7d547dff182)

(Ignore the black rectangles, that's the screenreader focus)

**Comment by LindsayYoung** [comment]
this is on dev 
![Screen Shot 2020-05-29 at 4.19.26 PM.png](https://images.zenhubusercontent.com/56f1892d6caf56c9d38e1ac3/28d7d952-8475-40a0-aba8-dcb5238923a9)

**Comment by Jacklynn** [comment]
Sorry, wrong screenshot. This is for the last confirmation page, not the landing page:

<img width="778" alt="Screen Shot 2020-05-28 at 3 03 38 PM" src="https://user-images.githubusercontent.com/2560618/83313098-9e075300-a1c9-11ea-875b-f765e29b1bc0.png">

**Comment by Jacklynn** [comment]
So I don't think the label should be there @LindsayYoung

**Comment by LindsayYoung** [comment]
Sorry for the confusion, I removed the label and put it in ready for dev

**Comment by Jacklynn** [comment]
I didn't see this in staging today, should I be checking somewhere else?

**Comment by cantsin** [comment]
Only in dev currently.

**Comment by blaughman** [comment]
Icons are next to links.  Moving to UAT.

**Comment by blaughman** [comment]
@Jacklynn - once your test is done, feel free to move this card along.

**Comment by blaughman** [comment]
Icons are present.

**Comment by lalitha-jonnalagadda** [comment]
Deployed on 6/16
